devise.rb

```diff
@@ -14,13 +14,15 @@
 
 inject_into_file 'Gemfile', after: 'group :development, :test do' do
   <<-RUBY
+
   gem 'pry-byebug'
   gem 'pry-rails'
   gem 'dotenv-rails'
```

Comment:
Perhaps we could take advantage of this refacto to add them in alphabetical order? 🤓

Reply:
Yes good idea 👍